packages/kit/test/load-nuxt-config.spec.ts

```diff
@@ -14,7 +14,6 @@ describe('loadNuxtConfig', () => {
         "#build": "<rootDir>/.nuxt",
         "#internal/nuxt/paths": "<rootDir>/.nuxt/paths.mjs",
         "#layers/layer-fixture": "<rootDir>",
-        "#layers/test": "<rootDir>/layers/test",
```

Comment:
it seems we have a behavior change we'll need to investigate

Reply:
fixed now!